pixart.js

```diff
@@ -1 +1,13 @@
+$(document).on("ready", function()
+{
+var setcolor=$("#set-color");
+setcolor.on("click", changeBoxToInput);
 
+var input=$(":input").val();
+
+
+function changeBoxToInput(){
+  $(".brush").css("background", ":input");
+  event.preventDefault()
+}
+});
```

Comment:
I'd urge you to revisit this if you have time, you'd probably be surprised at how much easier it would be to tackle it. The last couple of days have been jam packed, and you're probably leveled up a lot.